src/prefect/tasks/filehandling/operations.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+from typing import Union, Any
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+from prefect.tasks.filehandling.base import FileBase
+
+
+class Copy(Task):
+    pass
```

Comment:
Did you mean to remove this one? :)

Reply:
Ups...this is another one to come in the future. Will remove it for this PR